src/pixel-ping.coffee

```diff
@@ -29,7 +29,15 @@ serialize = ->
 
 # Reset the `store`.
 reset = ->
+  old_store = store
   store = {}
+  old_store
+
+# Merge the given `store` with the current one.
+merge = (new_store) ->
+  for key, hits of new_store
+    store[key] or= 0
```

Comment:
Wat. Having a hard time with this. coffee seems really foreign.

Reply:
Yeah. coffee isn't as much fun as it was purported to be.

Reply:
I assume this is equivalent to `store[key] = store[key] || 0`. We use this logic twice now. It could be abstracted into something like `increment(n)`.